README.md

```diff
@@ -107,30 +107,11 @@ sudo apt-get install llvm clang cmake
 
 #### macOS
 
-1. Install via Homebrew:
-
-```bash
-brew install llvm@15 clang cmake
-```
-
-2. Add `llvm` to your `~/.zshrc` or `~/.bashrc`:
-
-```bash
-export PATH="/opt/homebrew/opt/llvm@15/bin:$PATH"
-```
-
-3. Activate the changes:
-
-```bash
-source ~/.zshrc
-# or
-source ~/.bashrc
-```
-
-4. Verify that `llvm` is installed:
+Install via Homebrew:
```

Comment:
This is incorrect change. This describes how to build software from source, not how to install precompiled software using homebrew.

Also while updating this is fine, the user-facing docs are at https://docs.subspace.network/

Reply:
Understood!

Ok I will revert the change.

Then should we have a another section with homebrew installation for macOS under Binary in README? 

Like this:

```markdown
## Binary

### macOS

Install using homebrew:


```

Reply:
Yeah, I think that'll work.

cc @EmilFattakhov

Reply:
Fixed. [commit](https://github.com/subspace/pulsar/pull/295/commits/3deb63475a8e40037ceca881fb4068c21d27576b)